source/class/qx/html/Root.js

```diff
@@ -71,7 +71,7 @@ qx.Class.define("qx.html.Root",
      * @param elem {Element} the dom element to set
      * @throws {Error} if the element is assigned again
      */
-    useElement : function(elem)
```

Comment:
this seems to be a public method name ... why change it ?

Reply:
Only to better match the meaning (it's defined in `qx.html.Node`, and can be used to "use" text nodes, so "element" is a bit of a weird name); `useElement` still exists but is deprecated, and it is rare in the existing code to need to call it so i expect the change to be low impact

Reply:
ok